### DISCUSSION TITLE
るびま20周年に何かしたい

### DISCUSSION THREAD

**Discussion by neko314** [rubima/magazine.rubyist.net#494] [要望・アイデア]
20周年というのは https://magazine.rubyist.net/articles/0001/0001-index.htm が2004年9月に出たらしいのでそこから計算して

**Comment by hsbt** [comment]
アイデアメモ

* Hotlinks: 20周年なのでこの人、というインタビューをやる
* Hotlinks 2週目: 1回インタビューした人に20年たってどうですか、というインタビューをやる
* Ruby でこんなことをやってます、というテッキーな記事をあつめる
* Ruby feature: Namespace, HEv2 などの内部の細かいテッキーな話

**Reply by kakutani** [reply]
> * Hotlinks: 20周年なのでこの人、というインタビューをやる

いまるびまを頑張っている @neko314 が @ko1 にるびま20周年インタビューをする、というのはどうか!??!

> * Ruby でこんなことをやってます、というテッキーな記事をあつめる

"by name"で「書いてくれ〜」とお願いしにいくのがよさそう。

> * Ruby feature: Namespace, HEv2 などの内部の細かいテッキーな話

* Namespace: @taggomoris に書いてくれ〜とお願いしたけど8月頃はムリ〜とのことでした(落ち着いたら書いてもらいたいですね)
* HEv2: @shioimm 👀 

パーサー方面とか、IRB方面とか??

RBS方面は @soutaro のDefinitiveな解説記事がFindyさんとこに掲載されてしまった…
[Steep、RBSはいかにしてRubyに静的な型検査を持ち込むか（松本宗太郎さん寄稿文） \- Findy Engineer Lab](https://findy-code.io/engineer-lab/soutaro)

**Reply by neko314** [reply]
> いまるびまを頑張っている @neko314 が @ko1 にるびま20周年インタビューをする、というのはどうか!??!

笹田さんがご了承くださった場合に考えることかも知れませんが、
私が馬にいるのとか話のアウトラインのようなものを用意したりするのは是非是非やりたいです！（笹田さんにお願いのご連絡したりとかも 🙆‍♂️ ）

しかしながらHotlinksっぽさは気心知れた人たち同士でざっくばらんに話して意外なお話とか引き出す点にある気がするので、笹田さんの友達的な方やなんか気軽に話に突っ込める方々にいてほしいです😃いつもどうやって人が集まっているのだろう...?

**Reply by kakutani** [reply]
> * HEv2: @shioimm 👀

ご本人に聞いたところ、実装が「だいぶ変わる」ﾄﾉｺﾄなので、9月までにはキビシイんじゃないかということが判明した(ばたり)。

**Reply by neko314** [reply]
> Hotlinks: 20周年なのでこの人、というインタビューをやる
いまるびまを頑張っている @neko314 が @ko1 にるびま20周年インタビューをする、というのはどうか!??!


OKいただきました(嬉)

**Comment by kakutani** [comment]
考古学。10周年までは毎年、周年スペシャルが掲載されていた。9月〜遅れても10月上旬には号が出ている 👀 
基本構成は関係者から寄せられたコメント + 1年のまとめと展望というもの。
ちょうど10年、周年企画は途絶えていたことになるっぽい…!!
10年ぶりに「何かやる」のは賛成…!

* 一周年: [Rubyist Magazine 0009 号](https://magazine.rubyist.net/articles/0009/0009-index.html) (2005-09-06)
  * コメント集と展望(ko1)
* 二周年: [Rubyist Magazine 0016 号](https://magazine.rubyist.net/articles/0016/0016-index.html) (2006-09-20)
  * コメント集と展望(ko1)
* 三周年: [Rubyist Magazine 0021 号](https://magazine.rubyist.net/articles/0021/0021-index.html) (2007-09-29)
  * コメント集と展望(ko1)
* 四周年: [Rubyist Magazine 0024 号](https://magazine.rubyist.net/articles/0024/0024-index.html) (2008-10-01)
  * コメント集なし。展望(ko1)
* RubyKaigi 2009: [Rubyist Magazine が出来るまで \- 日本Ruby会議2009](https://rubykaigi.org/2009/ja/talks/18H04/) (2009-07-18)
  * 五周年を前にささださんにるびまの話をしてもらった
* 五周年: [Rubyist Magazine 0027 号](https://magazine.rubyist.net/articles/0027/0027-index.html) (2009-09-13)
  * コメント集と展望(ko1)
* 六周年: [Rubyist Magazine 0031 号](https://magazine.rubyist.net/articles/0031/0031-index.html) (2010-10-07)
  * コメント集と展望(ko1)
* 七周年: [Rubyist Magazine 0035 号](https://magazine.rubyist.net/articles/0035/0035-index.html) (2011-09-26)
  * コメント集と展望(ko1, sunaot)
* 八周年: [Rubyist Magazine 0039 号](https://magazine.rubyist.net/articles/0039/0039-index.html) (2012-09-05)
  * コメント集と展望(gunjisatoshi)
* 九周年: [Rubyist Magazine 0044 号](https://magazine.rubyist.net/articles/0044/0044-index.html) (2013-09-30)
  * コメント集と展望(gunjisatoshi)
* 十周年: [Rubyist Magazine 0048 号](https://magazine.rubyist.net/articles/0048/0048-index.html) (2014-09-19)
  * コメント集と展望(gunjisatoshi)、「あの記事を振り返って(zundan)」

**Reply by kakutani** [reply]
20周年コメントを集めるのはやりやすいのでよさそう。
広く募集するやつに加えて、10周年のやつみたいに対象を決めたコメントを集めるのもよさそう。「あの記事を振り返って」ではそれまでに寄稿した人たちに声をかけていた。

**Reply by neko314** [reply]
「10周年のやつみたいに対象を決めたコメント」とほぼ同じですが、広く募るのと、編集や執筆をした方で発刊当初〜最近の方までこちらからお願いしたりしてメッセージを書いていただけると良さそうだなーと思いました。（meetupでお会いしたので志村さんに直接そういうお話をしてみたところ、書いていただけるとのことでした🙂）

やり方としては
広く募る方ではRubyKaigiの時のようにissueにしてコメントとしてメッセージをいただく・それを随所で告知する、
お願いする方ではX（とは未だに言い慣れない）やSlack、メールなどでご連絡してみる、
というような方法がありそう。

後者についてはどなたにご連絡するのが良いかというのと、そもそも私個人が直接ご連絡とれる方の方が少なそうな気がしているので、どうやってお願いしていくかご相談したいです！

**Comment by neko314** [comment]
私にdiscussionの仕草が身についておらず放置してしまっていた。。。0064号の一部として20周年記念の記事をリリースすることができたのでこちらcloseします。皆さんありがとうございました！
